tendrl/gluster_integration/manager/__init__.py

```diff
@@ -34,10 +34,12 @@ def main():
     NS.state_sync_thread = sds_sync.GlusterIntegrationSdsSyncStateThread()
 
     NS.node_context.save()
-    NS.tendrl_context.save()
-    NS.gluster_integration.definitions.save()
-    NS.gluster_integration.config.save()
+    NS.gluster.tendrl_context.save()
+    NS.gluster.definitions.save()
+    NS.gluster.config.save()
     NS.publisher_id = "gluster_integration"
+    NS.tendrl_context = \
```

Comment:
this is not required, already done in https://github.com/Tendrl/commons/blob/develop/tendrl/commons/__init__.py#L70